### PR TITLE
HMS Rapid Redeploy Documentation and scripts CASMHMS-5772/CASMHMS-5773/CASMHMS-5774

### PR DESCRIPTION
**Pull request by mbuchmann-hpe**
# Description
Documentation and scripts for the Rapid Redeploy process (backup and restore).

Tested on Wasp as part of the process for implementation on UKMet

- CASMHMS-5772
- CASMHMS-5773
- CASMHMS-5774

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by shunr-hpe — APPROVED** [review]

File: operations/Backup_HMS.md
Comment (rustydb) [inline operations/Backup_HMS.md:1]: image

The indentation is breaking the ordered list.


For anything that's a part of a list item, it should be indented to at least the first character in the first line.

e.g.

````markdown
1. Lorem ipsom

   Another statement, aligned with "L"

1. Another step

   ```bash
   echo "hello world"
   ```

````

**Review by rustydb — CHANGES_REQUESTED** [review]

File: operations/Backup_HMS.md
Comment (mbuchmann-hpe) [reply operations/Backup_HMS.md:1]: Updated indentation

File: operations/hardware_state_manager/HSM_Backup_User_Data.md
Comment (rustydb) [inline operations/hardware_state_manager/HSM_Backup_User_Data.md:1]: The indentation breaks the ordered list and formatting.

image

For anything that's a part of a list item, it should be indented to at least the first character in the first line.

e.g.

````markdown
1. Lorem ipsom

   Another statement, aligned with "L"

1. Another step

   ```bash
   echo "hello world"
   ```

````

**Review by rustydb — CHANGES_REQUESTED** [review]
Indentation is broken, see comment.

**Review by rustydb — APPROVED** [review]

**Pull request merged**